kjourney/README.md

```diff
@@ -1,6 +1,8 @@
+## Angular material 
+https://www.freakyjolly.com/angular-material-7-8-add-expansion-panel-accordion-in-angular-project-using-material/#more-2551
 # Kjourney
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.17.
+This project waßs generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.17.
```

Comment:
Is this change necessary? Or is this a mistake/automatically changed?

Reply:
Letter has been deleted